README.md

```diff
@@ -40,7 +58,7 @@ distributed as the
 [`mongodb/mongodb`](https://packagist.org/packages/mongodb/mongodb) package for
 [Composer](https://getcomposer.org).
 
-## Release Integrity
+### Release Integrity
```

Comment:
Release integrity is relevant even when not installing with PECL, and moving it to a sub-header doesn't make sense IMO. We can add a general "Installation" header above and make the sections for pie and pecl sub-sections of that installation section.

Since pie doesn't support signature verification and also doesn't support installing from a downloaded binary, we should add a note that integrity validation is not yet supported with pie.